src/app/models/hardcoded-package-names.ts

```diff
@@ -0,0 +1,7 @@
+export const HARDCODED_PACKAGE_NAMES = [
```

Comment:
Are we cool with having constants live in `models` directory? Or do we want to put them somewhere else?

Reply:
https://www.becomebetterprogrammer.com/typescript-organizing-and-storing-types-and-interfaces/#Typed-based